flang/lib/Optimizer/Transforms/RaiseToAffine.cpp

```diff
@@ -9,40 +9,116 @@
 #include "PassDetail.h"
 #include "flang/Optimizer/Dialect/FIRDialect.h"
 #include "flang/Optimizer/Dialect/FIROps.h"
+#include "flang/Optimizer/Transforms/AffinePromotion.h"
 #include "flang/Optimizer/Transforms/Passes.h"
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/Visitors.h"
 #include "mlir/Dialect/Affine/IR/AffineOps.h"
 #include "mlir/Dialect/StandardOps/IR/Ops.h"
 #include "mlir/Pass/Pass.h"
 #include "mlir/Transforms/DialectConversion.h"
+#include "llvm/ADT/DenseMap.h"
 #include "llvm/Support/CommandLine.h"
+#define DEBUG_TYPE "flang-affine-promotion"
+#include <iostream>
```

Comment:
LLVM has a prohibition on `iostream`.
https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden

You can typically use llvm::errs() or llvm::outs() however.

Reply:
oops, I used llvm::dbgs() and forgot to remove iostream.